docs/contribute/modules/index.md

```diff
@@ -126,6 +127,9 @@ mkdocs serve
 
 ... then viewing the documentation using the link in your terminal.
 
+### Create a new branch of the parent package
+
```

Comment:
Depending on weather you are member of the AutoRA organisation, to create a pull request, you need to fork the package and create a pull request from there. If you are not a member of the organisation you can not just create a branch.

Reply:
good point! what about something like this?